libsnark/gadgetlib1/gadgets/hashes/anemoi/tests/test_anemoi_gadget.cpp

```diff
@@ -0,0 +1,221 @@
+/** @file
+ *****************************************************************************
+ * @author     This file is part of libsnark, developed by SCIPR Lab
+ *             and contributors (see AUTHORS).
+ * @copyright  MIT license (see LICENSE file)
+ *****************************************************************************/
+
+#include <gtest/gtest.h>
+#include <libff/algebra/curves/bls12_381/bls12_381_init.hpp>
+#include <libff/algebra/curves/bls12_381/bls12_381_pp.hpp>
+#include <libff/common/default_types/ec_pp.hpp>
+#include <libff/common/profiling.hpp>
+#include <libff/common/utils.hpp>
+#include <libsnark/common/default_types/r1cs_ppzksnark_pp.hpp>
+#include <libsnark/gadgetlib1/gadgets/hashes/anemoi/anemoi_components.hpp>
+#include <libsnark/gadgetlib1/gadgets/hashes/anemoi/anemoi_constants.hpp>
+#include <libsnark/zk_proof_systems/ppzksnark/r1cs_gg_ppzksnark/r1cs_gg_ppzksnark.hpp>
+//#include
+//<libsnark/zk_proof_systems/ppzksnark/r1cs_ppzksnark/r1cs_ppzksnark.hpp>
+
+using namespace libsnark;
+
+template<typename ppT>
+void test_pb_verify_circuit(protoboard<libff::Fr<ppT>> &pb)
```

Comment:
Seems unused.

Reply:
It should have been used in `test_flystel_prime_field_gadget`, but was commented for debug. Now fixed. Note that `test_pb_verify_circuit` is supposed to be called after each `ASSERT_TRUE(pb.is_satisfied());` . The missing calls will be added after `anemoi-hash-r1cs-parametrization` is merged into `anemoi-hash-r1cs` .

Reply:
#77

Reply:
Addressed.